keyboards/splitkb/kyria/rev1/rev1.h

```diff
@@ -20,6 +20,28 @@
 #    include "proton_c.h"
 #endif
 
+#ifdef OLED_ENABLE
+// clang-format off
+static const char PROGMEM qmk_logo[] = {
```

Comment:
Shouldnt be adding constants to header files, this should be reverted.

Reply:
Fixed, made it extern and non-static.